### PR TITLE
create brief introduction page for mobile website.

### PR DESCRIPTION
**Pull request by bengigo**
-changes made in this branch:

- set up HTML&CSS linters

- create a brief introduction page including
    - logo
    - navigation menu
    - introduction text
    - contact links
   
- use Figma template to clone a design

### CODE REVIEW

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @bengigo , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does NOT have a short summary of changes introduced in the pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  font-weight: 700;
+}
+
+#burger {
+  border: dotted;
+  width: 16px;
+  height: 10px;
+  margin-right: 24px;
+  margin-top: 15px;
+}
```
Comment (CollinsTatang) [inline styles.css:43]: - [ ] Your design looks good but you still have to adjust it again to meet [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
Your design: 
headerError
Figma design:
figmaDesign

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @bengigo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Gitflow was correctly used ✔️ 


## Required Changes ♻️

- [ ] Kindly add a descriptive PR with a summary of all files changes as requested by the previews reviewer

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bengigo** [comment]
@CollinsTatang thank you for your review. There is a part I'm not sure I clearly understand: "Kindly add a descriptive PR with a summary of all files changes as requested by the previews reviewer".
Couldn't be sure if I'm supposed to make another pull request with the proper description or edit the current pull request's description with the latest suggested changes I made.

**Comment by ShoiraTa** [comment]
> 
Since Collins did not answer yet, You were supposed to edit current PR as you have done, great job! 🙌🏻

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi team,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- No linter errors ✔️ 
- Amazing UI ✔️ 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**